src/xdocs/config_design.xml

```diff
@@ -1516,47 +1678,87 @@ public class ImmutableClass
 &lt;module name=&quot;VisibilityModifier&quot;&gt;
   &lt;property name=&quot;allowPublicImmutableFields&quot; value=&quot;true&quot;/&gt;
   &lt;property name=&quot;immutableClassCanonicalNames&quot;
-    value=&quot;com.google.common.collect.ImmutableSet, com.google.common.collect.ImmutableMap,
-      java.lang.String&quot;/&gt;
+    value=&quot;com.google.common.collect.ImmutableSet,
+          com.google.common.collect.ImmutableMap,
+          java.lang.String&quot;/&gt;
 &lt;/module&gt;
         </source>
         <p>
           Example of how the check works:
         </p>
         <source>
-public final class Test {
-  public final String s;
-  public final ImmutableSet&lt;String&gt; names;
-  public final ImmutableSet&lt;Object&gt; objects; // violation (Object class is mutable)
-  public final ImmutableMap&lt;String, Object&gt; links; // violation (Object class is mutable)
+public final class MyClass {
+ private int myPrivateField1;
 
-  public Test() {
-    s = "Hello!";
-    names = ImmutableSet.of();
-    objects = ImmutableSet.of();
-    links = ImmutableMap.of();
-  }
+ int field1;               // violation, must have a visibility modifier
+
+ protected String field2;  // violation, protected visibility is not allowed
+
+ public int field3 = 42;  // violation, not static final, immutable,
+                          // nor matching the pattern
+ public long serialVersionUID = 1L;
+ public static final int field4 = 42;
+
+ public final int field5 = 42;
+ public final java.lang.String notes;
+
+ public final Set&lt;String&gt; mySet1 = new HashSet&lt;&gt;();  // violation, HashSet is mutable
+ public final ImmutableSet&lt;String&gt; mySet2;
+ public final ImmutableMap&lt;String, Object&gt; objects1; // violation
+                                                     // 'Object' still considered as mutable
+
+ @com.annotation.CustomAnnotation
+ String annotatedString;      // violation
+
+ @CustomAnnotation
+ String shortCustomAnnotated; // violation
```

Comment:
Why is there no violation message for these?

Reply:
The annotation above it explains the difference, any message will just repeat what is already written.

Reply:
Let's keep examples consistent, please add a similar violation message to these too.

Reply:
@0xbakry , is it done?